pkgs/development/lua-modules/overrides.nix

```diff
@@ -456,4 +456,5 @@ with prev;
 
   # aliases
   cjson = prev.lua-cjson;
+  protobuf = prev.lua-protobuf;
```

Comment:
I would rather avoid the alias, the cjson one was for backwards compatibility.

Reply:
Okay, then I misinterpreted that one. Thought it was for convenience when using the pkg.
Updated.